Makefile.kokkos-kernels

```diff
@@ -435,6 +448,10 @@ ifeq (${KOKKOSKERNELS_INTERNAL_ENABLE_CUBLAS}, 1)
   KOKKOSKERNELS_INTERNAL_SRC_BLAS += ${KOKKOSKERNELS_PATH}/src/impl/tpls/KokkosBlas_Cuda_tpl.cpp
 endif
 
+ifeq (${KOKKOSKERNELS_INTERNAL_ENABLE_MAGMA}, 1)
+  KOKKOSKERNELS_INTERNAL_SRC_BLAS += ${KOKKOSKERNELS_PATH}/src/impl/tpls/KokkosBlas_Cuda_tpl.cpp
```

Comment:
Is this really necessary ? With CUDA enabled, cuda_tpl.cpp is included in the SRC. Magma cannot be enabled without CUDA.

Reply:
@kyungjoo-kim I think it is needed. With CUDA enabled, cuda_tpl.cpp is not automatically included in the SRC. It is only included if KOKKOSKERNELS_ENABLE_TPL_CUBLAS or KOKKOSKERNELS_ENABLE_TPL_MAGMA

Reply:
Okay. I see that.